agent.go

```diff
@@ -486,6 +486,11 @@ func (s *sandbox) listenToUdevEvents() {
 			continue
 		}
 
+		// We only care about add event
+		if uEv.Action != "add" {
+			continue
+		}
```

Comment:
Please put this in a separate commit, as this is not related to the uevent for memory.

Reply:
Why ask for splitting a commit into two? The checks **MUST** be separated otherwise I have to check it again when testing for memory online path.

Reply:
Ah ok I understand.